lib/sequel/adapters/drill.rb

```diff
@@ -80,6 +80,8 @@ def create_table_generator_class
 
     class Dataset < Sequel::Dataset
```

Comment:
This class extends the base Sequel classes, which we don't have control over. I'm just more-or-less following the sequel-vertica recipe here, using it as a general model.